From 89be3c1e1ad0ecd44d3a54d0780be45c45809668 Mon Sep 17 00:00:00 2001
From: Kas User <kas@example.com>
Date: Mon, 31 May 2021 21:09:49 -0300
Subject: [PATCH] Fix unsafe compilation path in the wx configure script.

Upstream-Status: Inappropriate [Yocto specific]
---
 lib/wx/configure | 29 -----------------------------
 1 file changed, 29 deletions(-)

diff --git a/lib/wx/configure b/lib/wx/configure
index 18c74090c2..29dffdf2d0 100755
--- a/lib/wx/configure
+++ b/lib/wx/configure
@@ -4529,35 +4529,6 @@ fi
 
 done
 
-       	if test X"$ac_cv_header_GL_gl_h" != Xyes ; then
-	    { $as_echo "$as_me:${as_lineno-$LINENO}: Checking for OpenGL headers in /usr/local" >&5
-$as_echo "$as_me: Checking for OpenGL headers in /usr/local" >&6;}
-	    CPPFLAGS="-isystem /usr/local/include $saved_CPPFLAGS"
-	    $as_unset ac_cv_header_GL_gl_h
-	    for ac_header in GL/gl.h
-do :
-  ac_fn_c_check_header_mongrel "$LINENO" "GL/gl.h" "ac_cv_header_GL_gl_h" "$ac_includes_default"
-if test "x$ac_cv_header_GL_gl_h" = xyes; then :
-  cat >>confdefs.h <<_ACEOF
-#define HAVE_GL_GL_H 1
-_ACEOF
-
-fi
-
-done
-
-	    if test X"$ac_cv_header_GL_gl_h" != Xyes ; then
-		{ $as_echo "$as_me:${as_lineno-$LINENO}: WARNING: No OpenGL headers found, wx will NOT be usable" >&5
-$as_echo "$as_me: WARNING: No OpenGL headers found, wx will NOT be usable" >&2;}
-		echo "No OpenGL headers found, wx will NOT be usable" >> ./CONF_INFO
-		WXERL_CAN_BUILD_DRIVER=false
-		CPPFLAGS="$saved_CPPFLAGS"
-	    else
-	    	GL_LIBS="-L/usr/local/lib $GL_LIBS"
-	    fi
-	else
-    	    GL_LIBS="-L/usr/X11R6/lib $GL_LIBS"
-       	fi
     fi
 else
     for ac_header in gl/gl.h
-- 
2.25.1

