From 3e656f302d34985dbdac598aa6f309044bec1635 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jo=C3=A3o=20Henrique=20Ferreira=20de=20Freitas?=
 <joaohf@gmail.com>
Date: Wed, 25 Mar 2020 23:39:02 +0100
Subject: [PATCH] Disable crosscompile before checking erlang native
 environment

---
 configure.ac | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/configure.ac b/configure.ac
index ddf7121..4ddac44 100644
--- a/configure.ac
+++ b/configure.ac
@@ -13,6 +13,9 @@ AC_SUBST([CONFIGURE_DEPENDENCIES],[vsn.mk])
 
 AC_PATH_PROG(SED, sed)
 
+tmp_cross_compiling=$cross_compiling
+cross_compiling=no
+
 AC_LANG(Erlang)
 AC_ARG_WITH(erlang, [  --with-erlang=PREFIX    path to erlc and erl ])
 AC_ERLANG_PATH_ERLC(erlc, $with_erlang:$with_erlang/bin:$PATH)
@@ -99,6 +102,7 @@ AC_CACHE_CHECK([new time API],
         [AC_MSG_RESULT(WARNING: new time API not available. use old now() instead)])
     ])
 
+cross_compiling=$tmp_cross_compiling
 
 AC_SUBST(erlang_cv_new_time_api)
 AC_SUBST(ERL_OPTS)
-- 
2.17.1

