From 405d88da526a402cbf272c477bf4d724e8a871c8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jo=C3=A3o=20Henrique=20Ferreira=20de=20Freitas?=
 <joaohf@gmail.com>
Date: Sun, 15 Oct 2023 21:30:38 -0300
Subject: [PATCH] Use autoconf 2.72c

Upstream-Status: Inappropriate [OE specific]

---
 otp_build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/otp_build b/otp_build
index 7b35b39fc2..8af65f4d8b 100755
--- a/otp_build
+++ b/otp_build
@@ -19,7 +19,7 @@
 # %CopyrightEnd%
 #
 
-USE_AUTOCONF_VERSION=2.71
+USE_AUTOCONF_VERSION=2.72c
 
 autoconf_aux_dirs="./erts/autoconf ./lib/common_test/test_server"
 
