From 60af90f8f0165fbe257ded7fe4e7c3269951f205 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jo=C3=A3o=20Henrique=20Ferreira=20de=20Freitas?=
 <joaohf@gmail.com>
Date: Wed, 14 Oct 2020 15:42:07 +0000
Subject: [PATCH] Fix release number

---
 rebar.config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rebar.config b/rebar.config
index a0600d55..ea215c76 100644
--- a/rebar.config
+++ b/rebar.config
@@ -45,7 +45,7 @@
     ]}
 ]}.
 
-{relx, [{release, {riak, "3.0"},
+{relx, [{release, {riak, "3.0.1"},
     [kernel,
      stdlib,
      lager,
-- 
2.25.1

